src/main/java/org/team1540/robot2024/Constants.java

```diff
@@ -16,7 +18,7 @@ public final class Constants {
     // Whether to pull PID constants from SmartDashboard
     public static final boolean tuningMode = true; // TODO: DO NOT SET TO TRUE FOR COMP
```

Comment:
this shouldn't be true on main

Reply:
more useful to have as true and set to false in a month than have to change every time we need to tune